### ISSUE TITLE
Feature Request: Show deleted user name in asset history

### ISSUE THREAD

**Issue by ernestasen** [snipe/snipe-it#2945]
It comes handy to see who was using the asset through time, but when I delete the user (an employee that doesn't work anymore) it shows "Deleted User".

I would like to see exactly who was using it because the asset below was used by two users who are deleted now, so I will loose track of who used it. This is required for our accounting company to know who was financially responsible for the equipment during a certain period of time.

![screen shot 2016-11-18 at 11 09 07](https://cloud.githubusercontent.com/assets/5003110/20424686/cc066724-ad7f-11e6-90dc-75308f0178b8.png)

**Comment by snipe** [comment]
@ernestasen That already happens, assuming you haven't purged data.

```
@if ($log->target->deleted_at=='')
                                <a href="{{ route('view/user', $log->target_id) }}">
                                {{ $log->target->fullName() }}
                                </a>
                              @else
                                <del>{{ $log->target->fullName() }}</del>
                              @endif
```

If you've purged data, there is no way to show the name of the deleted user, since they've been hard-deleted.

**Comment by ernestasen** [comment]
It happens immediatly after I delete the user. It just disappears from all asset logs. No need to purge.

**Comment by dmeltzer** [comment]
Which place are you looking at the history? The code has been different for a while depending on where you look at it--we're working to centralize it now.

**Comment by ernestasen** [comment]
I select the asset and then History tab:
<img width="1013" alt="screen shot 2017-01-03 at 15 32 17" src="https://cloud.githubusercontent.com/assets/5003110/21609239/e7b3f0a4-d1c9-11e6-9c56-2935deb71108.png">

**Comment by dmeltzer** [comment]
I'm pretty sure this is now fixed on develop.

**Comment by snipe** [comment]
That said, develop is not currently stable, so please don't use it on a production system.

**Comment by rocketnova** [comment]
This appears to not yet be fixed in 3.6.6. It looks like there's a lot of development activity for a new version that includes the fix. In the meantime, if it helps anyone, I found the issue to be resolved by changing [line 81 of app/Models/Actionlog.php](https://github.com/snipe/snipe-it/blob/v3.6.6/app/Models/Actionlog.php#L81) from `return $this->morphTo('target');` to `return $this->morphTo('target')->withTrashed();`
